src/pages/projectSettings/tabs/EventLogTab/EventLogTab.tsx

```diff
@@ -38,11 +47,121 @@ export const EventLogTab: React.FC = () => {
     },
   });
 
+  type logEntry = {
+    timestamp: Date;
+    user: string;
+    before?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+    after?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+  };
+
+  const eventData = isRepo
+    ? repoEventData?.repoEvents?.eventLogEntries
+    : projectEventData?.projectEvents?.eventLogEntries;
+
+  return (
+    <div data-cy="event-log">
+      {((eventData as logEntry[]) || []).map(
+        ({ user, timestamp, before, after }) => (
+          /* @ts-expect-error */
+          <EventLogCard key={`event_log_${timestamp}`}>
+            <EventLogHeader user={user} timestamp={timestamp} />
+            <Table
+              data={getEventDiffLines(before, after)}
+              columns={[
+                <TableHeader
+                  key="key"
+                  label={<span data-cy="key">Property</span>}
+                  sortBy={(datum: EventDiffLine) => datum.key}
+                />,
+                <TableHeader
+                  key="before"
+                  label={<span data-cy="before">Before</span>}
+                  sortBy={(datum: EventDiffLine) => datum.before}
+                />,
+                <TableHeader
+                  key="after"
+                  label={<span data-cy="after">After</span>}
```

Comment:
[nit] You can apply these `data-cy` props to the TableHeader components instead of adding spans